Evidently integration (#4918)
iterative/dvc.org: content/docs/user-guide/integrations/evidently.md

# Evidently

> DVCLive can be used to track the results

Reviewer: ```suggestion
[DVCLive](/doc/dvclive) can be used to track the results
```

Author: done

> of[Evidently](https://www.evidentlyai.com/). In the following we demonstrate it

Reviewer: ```suggestion
of [Evidently](https://www.evidentlyai.com/). In the following we demonstrate it
```

Author: done

```

You can adapt what you want to calculate by selecting a different Preset or
Metric from those available in Evidently.

## Define the comparison windows

Specify the period that is considered reference: Evidently will use it as the
base for the comparison. Then, you should choose the periods to treat as
experiments. This emulates the production model runs.

```python
#set reference dates
reference_dates = ('2011-01-01 00:00:00','2011-01-28 23:00:00')

#set experiment batches dates
experiment_batches = [
    ('2011-01-01 00:00:00','2011-01-29 23:00:00'),
    ('2011-01-29 00:00:00','2011-02-07 23:00:00'),
    ('2011-02-07 00:00:00','2011-02-14 23:00:00'),
    ('2011-02-15 00:00:00','2011-02-21 23:00:00'),
]
```

## Run and log experiments in DVC

There are two ways to track the results of Evidently with DVCLive:

1. you can save the results of each item in the batch in one single experiment
   (each experiment corresponds to a git commit), in separate steps
2. or you can save the result of each item in the batch as a separate experiment

We will demonstrate both, and show you how to inspect the results regardless of
your IDE. However, if you are using VSCode, we recommend using

> [our awesome VSCode extension](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)

Reviewer: ```suggestion
the [DVC extension for VS Code](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
```

Appreciate the love but technically this is what we are supposed to call it!

Author: done

# Evidently

[DVCLive](/doc/dvclive) can be used to track the results of
[Evidently](https://www.evidentlyai.com/). In the following we demonstrate it
through an example.

<p align='center'>

>   <a href="https://colab.research.google.com/drive/14usegPOSArF9tdO7NUOPndNk5vuUaYaX?usp=sharing">

Reviewer: ```suggestion
  <a href="https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLive-Evidently.ipynb">
```
